.travis.yml

```diff
@@ -38,8 +41,19 @@ matrix:
         packages:
         - texlive-latex-extra
         - texlive-fonts-recommended
+  - env: CI_KIND=build XARCH=s390x
+    arch: s390x
+    addons:
+      apt:
+        packages:
+        - texlive-latex-extra
+        - texlive-fonts-recommended
   - env: CI_KIND=build XARCH=x64 CONFIG_ARG=--disable-shared
   - env: CI_KIND=build XARCH=x64 MIN_BUILD=1
+  - env: CI_KIND=build XARCH=s390x CONFIG_ARG=--disable-shared
+    arch: s390x
+  - env: CI_KIND=build XARCH=s390x MIN_BUILD=1
+    arch: s390x
```

Comment:
There's no need for either of these tests - the `MIN_BUILD` and `--disable-shared` tests ensure that a section of the build system works correctly, it's not necessary to run them on multiple architectures